docs/docs/reference-react.md

```diff
@@ -0,0 +1,149 @@
+---
+id: reference-react
+title: React Top-Level API
+permalink: docs/reference-react.html
+next: reference-react-dom.html
+redirect_from: "/docs/reference.html"
+---
+
+`React` is the entry point to the React library. If you're using one of the prebuilt packages it's available as a global; if you're using CommonJS modules you can `require()` it.
```

Comment:
Many people these days don't know what CommonJS is, and beginners who learned with Babel don't know what `require()` is because they only saw `import`.